calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -860,6 +976,22 @@ def process_detection_results(self, img: np.array, detection: BoardDetection, im
             self.single_shot_reprojection_error_max_label.setText("Reprojection error (max):")
             self.single_shot_reprojection_error_avg_label.setText("Reprojection error (avg):")
             self.single_shot_reprojection_error_rms_label.setText("Reprojection error (rms):")
+            board_speed = None
+            self.image_view.set_draw_indicators(
+                board_speed,
+                self.data_collector.max_allowed_pixel_speed.value,
+                self.data_collector.get_skew_percentage(),
+                self.data_collector.get_size_percentage(),
+                0,
```

Comment:
you use some both 0.0 and 0 for the parameters, but at least the method uses float typing.
Please make it clear here. Also, since there are many parameters, consider using `name_parameter=value`

Reply:
ok i will use this structure

`name_parameter=value`

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e